src/autocomplete.js

```diff
@@ -490,36 +572,49 @@ class Autocomplete extends Component {
     // apply the props for them.
     this.getRootProps.called = false
     const {
+      // we need to pick off a bunch of variables from this.props
+      // because we only want to forward some of them.
       children,
-      // eslint-disable-next-line no-unused-vars
+      defaultHighlightedIndex,
       defaultValue,
-      // eslint-disable-next-line no-unused-vars
-      getValue,
-      // eslint-disable-next-line no-unused-vars
       getA11yStatusMessage,
-      // eslint-disable-next-line no-unused-vars
-      defaultHighlightedIndex,
-      // eslint-disable-next-line no-unused-vars
+      getValue,
       multiple,
-      // eslint-disable-next-line no-unused-vars
-      onClick,
-      // eslint-disable-next-line no-unused-vars
       onChange,
+      onClick,
+      // things from state that you _can_ control from the outside
+      // in a "controlled" component manner if you need to
+      selectedValue,
+      isOpen,
+      inputValue,
+      highlightedIndex,
       ...rest
     } = this.props
-    const element = children(this.getControllerStateAndHelpers())
-    if (this.getRootProps.called) {
+    // doing React.Children.only for Preact support ⚛️
```

Comment:
So I may have thrown in one or two extras in here (cc @zouhir @developit) :wink: